### PR TITLE
Added test cases for list-icon

### PR DESCRIPTION
**Pull request by Srinivas9933**
Added Test cases for list-icon

### CODE REVIEW

File: app/javascript/components/list-icon/component.spec.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { mountedComponent, screen } from "../../test-utils";
+
+import ListIcon from "./component";
+
+describe("<ListIcon />", () => {
+  it("renders correct icon", () => {
+    mountedComponent(<ListIcon icon="cases" />);
+    expect(screen.getByText((content, element) => element.tagName.toLowerCase() === "svg")).toBeInTheDocument();
+  });
+  it("renders incidents icon", () => {
```
Comment (jtoliver-quoin) [inline app/javascript/components/list-icon/component.spec.js:10]: Add a new line above

File: app/javascript/components/list-icon/component.spec.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { mountedComponent, screen } from "../../test-utils";
+
+import ListIcon from "./component";
+
+describe("<ListIcon />", () => {
+  it("renders correct icon", () => {
+    mountedComponent(<ListIcon icon="cases" />);
+    expect(screen.getByText((content, element) => element.tagName.toLowerCase() === "svg")).toBeInTheDocument();
```
Comment (jtoliver-quoin) [inline app/javascript/components/list-icon/component.spec.js:8]: For these, is it possible to add a test-id with the value of the icon prop in the ListIcon component and check the value of the test-id instead. Checking if a svg returns does not tell us which icons was returned.

File: app/javascript/components/list-icon/component.spec.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  });
+  it("renders incidents icon", () => {
+    mountedComponent(<ListIcon icon="incidents" />);
+    expect(screen.getByText((content, element) => element.tagName.toLowerCase() === "svg")).toBeInTheDocument();
```
Comment (jtoliver-quoin) [inline app/javascript/components/list-icon/component.spec.js:12]: Here also

**Review by jtoliver-quoin — CHANGES_REQUESTED** [review]

File: app/javascript/components/list-icon/component.spec.js (hunk 1 above)
Comment (Srinivas9933) [reply app/javascript/components/list-icon/component.spec.js:11]: Done

File: app/javascript/components/list-icon/component.spec.js (hunk 2 above)
Comment (Srinivas9933) [reply app/javascript/components/list-icon/component.spec.js]: Done

File: app/javascript/components/list-icon/component.spec.js (hunk 3 above)
Comment (Srinivas9933) [reply app/javascript/components/list-icon/component.spec.js]: Done

**Pull request merged**